Updates csv 13-05

sim55649 opened the pull request:
edit CHN , DNK , FRA (lv2) , GBR , IND , src

# What type of PR is this?

Select all that apply: [ ] -> [x]

- [ ] R 
- [x] CSV
- [ ] other

## R

I confirm: 

- [x] I checked the data
- [x] I checked the format with the `still_to_do` function
- [x] I read the [dataset description](https://covid19datahub.io/articles/doc/data.html)

## CSV

I confirm: 

- [x] I checked the data
- [x] I read the [dataset description](https://covid19datahub.io/articles/doc/data.html)

## Additional comments

Insert additional comments here...

sim55649 committed:
Updates csv 13-05

edit CHN , DNK , FRA (lv2) , GBR , IND , src

Commit:
Merge branch 'master' into updates_ISO

Commit:
fix FRA

eguidotti commented:
Thanks! I fixed FRA.
It seems that the sources for CHN, GBR, IND, etc.. are missing. Would you mind to check?

sim55649 committed:
Update 15-05 csv

IND already added
CHN ref on src added
GBR ref on src added
LVA complete update src+csv

sim55649 commented:
> Thanks! I fixed FRA.
> It seems that the sources for CHN, GBR, IND, etc.. are missing. Would you mind to check?

should have been correct in the commit I just made :)

Commit:
Merge branch 'master' into updates_ISO
